frontend/public/components/monitoring/targets.tsx

```diff
@@ -293,9 +320,34 @@ const REDUX_ID = 'monitoringTargets';
 
 const getRowProps = (target: Target) => ({ id: target.scrapeUrl, title: target.lastError });
 
+const getWatchError = (monitorsWatchContext, t) => {
+  const loadErrorIndex = 2;
+  const watchError = monitorsWatchContext[loadErrorIndex];
+  const authInfo =
+    'https://docs.openshift.com/container-platform/4.11/monitoring/enabling-monitoring-for-user-defined-projects.html';
+
+  if (watchError) {
+    if (watchError.code === 403) {
+      return (
+        <>
+          <p> {watchError.message} </p>
+          <ExternalLink href={authInfo} text={t('Learn More')} />
+        </>
+      );
+    }
+    return `${watchError.message}`;
+  }
+};
+
 const List: React.FC<ListProps> = ({ loaded, loadError, targets }) => {
   const { t } = useTranslation();
 
+  const [showServiceMonitorError, setShowServiceMonitorError] = React.useState(true);
+  const [showPodMonitorError, setShowPodMonitorError] = React.useState(true);
+
+  const serviceMonitorWatchError = getWatchError(React.useContext(ServiceMonitorsWatchContext), t);
+  const podMonitorWatchError = getWatchError(React.useContext(PodMonitorsWatchContext), t);
```

Comment:
```suggestion
  const [, , serviceMonitorWatchError] = React.useContext(ServiceMonitorsWatchContext);
  const [, , podMonitorWatchError] = React.useContext(PodMonitorsWatchContext);
```

Not an absolute rule, but generally functions that return DOM are less obvious, so I think it's better to just deal with the error string here and leave all the DOM construction until the `return`.